### PR TITLE
solve survey badge bug and some minor issues

### PR DESCRIPTION
**Pull request by kandelchy**

### CODE REVIEW

**Comment by ssharba** [comment]
conflicts

File: app/views/attempts/index.html.haml [hunk 1]
```diff
@@ -25,17 +25,17 @@
           %th % NO
       %tbody
         - @ads.each do |ad|
-          - t1 = @attempts.where(ad_id: ad.id, answer_id: 1).count.to_f
-          - t2 = @attempts.where(ad_id: ad.id, answer_id: 2).count.to_f
-          - t3 = @attempts.where(ad_id: ad.id, answer_id: 3).count.to_f
-          - t4 = @attempts.where(ad_id: ad.id, answer_id: 4).count.to_f
+          - t1 = @attempts.where(survey_form_id: 1, answer_id: 1).count.to_f
```
Comment (ssharba) [inline app/views/attempts/index.html.haml]: why id=1?

File: app/controllers/survey_forms_controller.rb [hunk 2]
```diff
@@ -10,7 +10,8 @@ def index
       # @survey_forms = SurveyForm.all
       @survey_forms = SurveyForm.where(sub_type: 2, is_published: false)
     elsif current_business
-      @survey_forms = SurveyForm.where(surveyable_type: 'Business', surveyable_id: current_business.id)
+      #  Need to tell Shahad about it
```
Comment (MwMaki) [inline app/controllers/survey_forms_controller.rb]: This Note will go to dev branch 😆

File: app/views/attempts/index.html.haml (hunk 1 above)
Comment (kandelchy) [reply app/views/attempts/index.html.haml]: This is an old report I crated for the Admin survey, which is the first survey we have, so its restricted to one specific survey

File: app/controllers/survey_forms_controller.rb (hunk 2 above)
Comment (kandelchy) [reply app/controllers/survey_forms_controller.rb]: So I cant leave a reminding comment in my code ?

**Pull request merged**